src/main/java/tombenpotter/sanguimancy/api/soulCorruption/SoulCorruption.java

```diff
@@ -3,12 +3,13 @@
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ResourceLocation;
 import net.minecraft.world.World;
 import net.minecraftforge.common.IExtendedEntityProperties;
+import tombenpotter.sanguimancy.Sanguimancy;
 
 public class SoulCorruption implements IExtendedEntityProperties {
-
-    public static String ID = "SoulCorruption";
```

Comment:
This should probably now be a capability, but the tutorial I was following was particularly unhelpful